CONTRIBUTING.md

```diff
@@ -50,6 +50,20 @@ You're now ready to go! You can now activate Storefront in your WordPress instal
 1. *Manually* - Each time you change a css file run `npm run css` in Terminal to compile all of the css.
 2. *Automatically* - run `npm run watch` in Terminal. This will instruct grunt to 'watch' each of the sass files for changes and then compile automatically when changes are made. This is the recommended method and will also watch Javascript files for changes too.
 
+##### NPM commnands
+
+Storefront utilizes npm scripts for task management utilities.
+
+- `npm run build`: Runs the build script
+- `npm run css`: Renders the SCSS files to CSS files
+- `npm run watch`: Watches changed of teh SCSS files and renders the CSS files accordingly
```

Comment:
Spelling:

```suggestion
- `npm run watch`: Watches changed of the SCSS files and renders the CSS files accordingly
```

Reply:
Oh boy, my signature typo. 🤦‍♂️ This is fixed as well now.